### ISSUE TITLE
Could not find meson version 0.63.3

### ISSUE THREAD

**Issue by hermannw** [bkbilly/lnxlink#61]
### What type of installation are you running?

Desktop

### Which Linux OS are you using?

Linux Mint Mate

### Which version of LNXLink has the issue?

latest

### The problem

Hi, I have a Dell tablet with a brand new installation of Linux Mint Mate.
I followed the installation steps from this page https://bkbilly.gitbook.io/lnxlink/setup. When I get to the step 6, everything works fine until it gets here:
```
Collecting dbus-python>=1.3.2
  Using cached dbus-python-1.3.2.tar.gz (605 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [3 lines of output]
      
      meson-python: error: Could not find meson version 0.63.3 or newer, found 0.61.2
      meson-python: error: .
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
hermann@hermann-Latitude-5175:~$ meson --version
1.1.1
```
First time I checked meson --version it said indeed 0.61.2 then I ran 
```pip3 install --update meson``` 
as recommended on the meson page and now it shows version 1.1.1, but the installation still fails with the same error. 
Any ideas what went wrong?

**Comment by bkbilly** [comment]
You can find the same issue on a previously closed issue #59
I'll update the installation instructions with the fix.

**Comment by hermannw** [comment]
Thanks! That worked.
Thank you also very much for sharing this software!!
